benchmarks/benchmarks/stats_sampling.py

```diff
@@ -74,6 +83,9 @@ def __repr__(self):
 #          \  0        otherwise
 # Taken from UNU.RAN test suite (from file t_pinv.c)
 class contdist4:
+    def __init__(self):
+        self.mode = 0
```

Comment:
There was a feature request to add `mode` to all PDFs. I suppose this would help in this case.